src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj

```diff
@@ -5,7 +5,7 @@
     <VersionPrefix>3.3.1</VersionPrefix>
```

Comment:
This change would require a major version bump, e.g. to 4.0.0

Reply:
You mean, because of dropping the `net451` and `net461` target frameworks? I have read both [.NET fundamentals ⟩ Library guidance ⟩ Breaking changes](https://docs.microsoft.com/en-us/dotnet/standard/library-guidance/breaking-changes) and [.NET fundamentals ⟩ Changes that affect compatibility](https://docs.microsoft.com/en-us/dotnet/core/compatibility/) which give very good advice but none of them mentions dropping a target framework.

Reply:
Dropping a framework will mean that users on .NET 4.5 (for example) won't be able to install the package, so it'll definitely be a breaking change from a SemVer perspective. I think because it's a big/deliberate breakage it's not really in the scope of what those docs are discussing. Let me know if I'm missing something here, though.

Reply:
Yeah, you're totally right. I just tried to add those properties in the `Serilog.Settings.Configuration.csproj` file in order to perform [package validation](https://docs.microsoft.com/en-us/dotnet/fundamentals/package-validation/overview):

```xml
<PropertyGroup>
  <EnablePackageValidation>true</EnablePackageValidation>
  <PackageValidationBaselineVersion>3.3.0</PackageValidationBaselineVersion>
</PropertyGroup>
```

Indeed, this is considered a breaking change. Here's the error after running `dotnet pack`:
> /usr/local/share/dotnet/sdk/6.0.202/Sdks/Microsoft.NET.Sdk/targets/Microsoft.NET.Compatibility.Common.targets(32,5): error PKV006: Target framework .NETFramework,Version=v4.5.1 is no longer supported in the latest version. [serilog-settings-configuration/src/Serilog.Settings.Configuration/Serilog.Settings.Configuration.csproj]

Reply:
I think unless as it doesn't cost anything from maintanence perspective point of view we can still support older TFM's even if they are eol. Bumping to 4.0 ideally would be good to be done with having some new features first, so it would not took long time to deliver stable release.

Reply:
I rebased on the `dev` branch, did the EOL target framework cleanup but only for the tests and sample projects in order not to introduce a breaking change.